soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4239,88 +4242,127 @@ void DrawRandoEditor(bool& open) {
                         "in the base pool."
                     );
                     SohImGui::EnhancementCombobox("gRandomizeIceTraps", randoIceTraps, 5, 1);
-                    ImGui::Separator();
                     ImGui::PopItemWidth();
                     ImGui::EndTable();
                 }
+                ImGui::PopStyleVar(1);
                 ImGui::EndTabItem();
             }
 
-            if (ImGui::BeginTabItem("Detailed Logic")) {
-                if (ImGui::BeginTable("tableRandoDetailedLogic", 3,
+            if (ImGui::BeginTabItem("Location Logic")) {
+                ImGui::PushStyleVar(ImGuiStyleVar_CellPadding, cellPadding);
+                if (ImGui::BeginTable("tableRandoLocationLogic", 2,
                                       ImGuiTableFlags_BordersH | ImGuiTableFlags_BordersV)) {
+                    ImGui::TableSetupColumn("Exclude Item Checks", ImGuiTableColumnFlags_WidthStretch, 200.0f);
                     ImGui::TableSetupColumn("Exclude Locations", ImGuiTableColumnFlags_WidthStretch, 200.0f);
```

Comment:
it's not clear what exclude locations vs exclude item checks are

i don't expect us to have "exclude fire temple" as an option (if that's what you were thinking by exclude locations)

also, in 64rando all the locations are listed in "exclude locations," i feel like "exclude item checks" makes things a little more confusing

Reply:
i think it's just "item" that's throwing me off, if it was just "exclude checks" that would be more clear imo

Reply:
![image](https://user-images.githubusercontent.com/4244591/182359939-4a17c6f7-3f16-4e49-95f2-89a53da0fd60.png)
I saw these before and thought it was actually excluding entire areas. But looking into it more, these might just be groups of locations that hold other individual locations.

That said, I'm not entirely sure what do change it to then. Should we just have 2 columns (or maybe even 3) without header names that both have locations to exclude in them?

Reply:
i'd say for now just leave the second column blank and remove the "coming soon"

once we have a better sense of how we want to handle the UX for managing hundreds of locations we can figure something else out